bin/packages/build.js

```diff
@@ -98,8 +99,17 @@ function buildStyle( packagePath ) {
 			.then( ( result ) => callback( null, result ) );
 	};
 
+	const postCSSRTLSync = ( ltrCSS, callback ) => {
```

Comment:
Should we merge #8093 and refactor to use the asynchronous code?

Reply:
I don't really care which one goes first but yeah, we should align